lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -210,8 +210,10 @@ defmodule ExUnit.DocTest do
 
   @doc false
   def __file__(module) do
-    source = module.__info__(:compile)[:source] ||
-             raise "#{inspect module} does not have compile-time source information"
+    source =
+      module.__info__(:compile)[:source] ||
+        raise "#{inspect(module)} does not have compile-time source information"
+
```

Comment:
why the newline?

Reply:
If an expression takes multiple lines, we always add a new line before and after.